playwright-typescript-playwright-test-main/playwright-typescript-playwright-test-main/testConfig.ts

```diff
@@ -1,13 +1,15 @@
 export const testConfig = {
     dev: ``,
-    username: `dotnetemployee@intonenetworks.com`,
-    password: `Password1!`,
+    url:'https://hrmsdev1.azurewebsites.net/',
+    username1: `dotnetemployee@intonenetworks.com`,
```

Comment:
Use Proper  Naming Conversions instead off numbers(1,2...etc)

Reply:
I given proper names instead of username1,Password1